protecode/model.py

```diff
@@ -81,3 +81,12 @@ def cve_major_severity(self) -> int:
 class ScanResult(ModelBase):
     def name(self):
         return self.raw.get('name')
+
+    def is_stale(self):
+        # Results are 'stale' if there has been a protecode-engine update since the last scan, i.e.
+        # a rescan might find more vulnerabilities
+        return self.raw.get('is_stale')
+
+    def has_binary(self):
+        # Indicates whether the binary has already been deleted on the protecode side
```

Comment:
nit1: this way it _could_ be misinterpreted that True means the file has been removed
nit2 (also for the previous hunk): why not format this as a docstring?